src/components/CountCard.vue

```diff
@@ -1,35 +1,38 @@
 <template>
-  <vx-card class="overflow-hidden" content-color="#fff" :card-background="background">
+  <vx-card class="card" content-color="#fff" :card-background="background">
     <div slot="no-body">
-      <div
-        class="p-6 pb-0"
-        :class="{'flex justify-between flex-row-reverse items-center': iconRight}"
-      >
-        <feather-icon
-          :icon="icon"
-          class="p-3 inline-flex rounded-full"
-          :class="[`text-$white`, {'mb-4': !iconRight}]"
-        ></feather-icon>
-        <div>
-          <h1 style="color:white;font-size:40px;" class="mb-1 fold-bold">{{statistic}}</h1>
-        </div>
+      <div class="p-6 pb-0 flex justify-between flex-row-reverse items-center">
+        <!--possible icons-->
+        <check-circle-icon v-if="index==0" class="feathericon"></check-circle-icon>
+        <x-circle-icon v-else-if="index==1" class="feathericon"></x-circle-icon>
+        <slash-icon v-else-if="index==2" class="feathericon"></slash-icon>
+        <alert-triangle-icon v-else-if="index==3" class="feathericon"></alert-triangle-icon>
```

Comment:
This is the kind of thing you'd do with a dirt simple functional component, if it was at all more complicated.

Reply:
By this i mean doing a selection from many html tags and icons based off a key :index in this case.